doc/classes/AudioStream.xml

```diff
@@ -28,6 +28,12 @@
 			<description>
 			</description>
 		</method>
+		<method name="_get_parameter_list" qualifiers="virtual const">
+			<return type="Array" />
```

Comment:
```suggestion
			<return type="Dictionary[]" />
```

Based on the failing CI prompt.

Reply:
Might make sense to also slightly tweak the description below after this change since "array contains dictionaries" will be redundant.